app/controllers/api/v1/notes_controller.rb

```diff
@@ -0,0 +1,54 @@
+module Api
+  module V1
+    class NotesController < ApplicationController
+      def index
+        render json: notes_filtered, status: :ok, each_serializer: IndexNoteSerializer
```

Comment:
Similar al comment del show, por lo general vas a encontrar simplemente `notes`
```suggestion
        render json: notes, status: :ok, each_serializer: IndexNoteSerializer
```